src/frontend/src/eth/components/wallet-connect/WalletConnectSession.svelte

```diff
@@ -336,12 +338,23 @@
 	$: walletConnectPaired.set(nonNullish(listener));
 
 	onDestroy(() => walletConnectPaired.set(false));
+
+	const openWalletConnectAuth = async () => {
+		modalStore.openWalletConnectAuth();
+
+		await trackEvent({
+			name: TRACK_COUNT_WALLET_CONNECT_MENU_OPEN
+		});
+	};
 </script>
 
 {#if nonNullish(listener)}
 	<WalletConnectButton on:click={disconnect}
 		>{$i18n.wallet_connect.text.disconnect}</WalletConnectButton
 	>
+{:else}
```

Comment:
@AntonioVentilii-DFINITY How do you test the wallet connect functionality locally?

Reply:
I generally connect with https://app.uniswap.org/